packages/manager/apps/pci-block-storage/src/pages/list/List.page.tsx

```diff
@@ -63,25 +56,8 @@ export default function ListingPage() {
 
   const { pagination, setPagination, sorting, setSorting } = useDataGrid();
 
-  useEffect(() => {
-    navigation
-      .getURL('public-cloud', `#/pci/projects/${projectId}`, {})
-      .then((data) => {
-        setProjectUrl(data as string);
-      });
-  }, [projectId, navigation]);
-
-  const {
-    data: allVolumes,
-    isFetching: isFetchingllVolumes,
-    isPending: isPendingAllVolumes,
-  } = useAllVolumes(projectId);
-  const {
-    data: volumes,
-    isLoading: isVolumesLoading,
-    isPending: isVolumesPending,
-    error,
-  } = useVolumes(
+  const { data: allVolumes } = useAllVolumes(projectId);
+  const { data: volumes, error, isFetching, isPending, refetch } = useVolumes(
```

Comment:
I'm a bit confused with the naming as, at first reading, I cannot distinguish the difference between `volumes` and `allVolumes`